### PR TITLE
Handling multiple subscriptions

### PR DESCRIPTION
**Pull request by 20shivangi**

### CODE REVIEW

File: Tasks/AzurePowerShellV4/InitializeAz.ps1 [hunk 1]
```diff
@@ -69,6 +95,16 @@ if ($endpointObject.scheme -eq 'ServicePrincipal') {
         throw (New-Object System.Exception("There was an error with the service principal used for the deployment.", $_.Exception))
        
     }
+
+    if($scopeLevel -eq "Subscription")
+    {
+        $SubscriptionId = $endpointObject.subscriptionId
+        $TenantId = $endpointObject.tenantId
+        $additional = @{ TenantId = $TenantId }
```
Comment (Ajay-MS) [inline Tasks/AzurePowerShellV4/InitializeAz.ps1]: We have one argument with splatting and other is without splatting. Is it intentional ?

Comment (20shivangi) [reply Tasks/AzurePowerShellV4/InitializeAz.ps1]: It is handled in the same way as used in other places.

File: Tasks/AzurePowerShellV4/InitializeAz.ps1 [hunk 2]
```diff
@@ -48,6 +48,32 @@ $null = Clear-AzContext -Scope Process
...
+
+    # Collect the parameters (names and values) in an array.
+    $parameters = foreach ($key in $Hashtable.Keys) {
+        $value = $Hashtable[$key]
+        # If the value is a bool, format the parameter as a switch (ending with ':').
+        if ($value -is [bool]) { "-$($key):" } else { "-$key" }
+        $value
+    }
+
+    $OFS = " "
```
Comment (kmkumaran) [inline Tasks/AzurePowerShellV4/InitializeAz.ps1]: What is this for?

Comment (20shivangi) [reply Tasks/AzurePowerShellV4/InitializeAz.ps1]: removed now

**Pull request merged**